src/client/CodeInput.js

```diff
@@ -45,14 +45,15 @@ var CodeInput = React.createClass({
         target.setSelectionRange(0, 10);
     },
     render() {
-        let { values } = this.state;
+        const { values } = this.state;
+        const { maxLen } = this.props;
         return (
             <div className="multi-input">
-                <input type="text" autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[0] = c} value={values[0]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(0, event)}/>
-                <span className="dash">-</span>
-                <input type="text" autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[1] = c} value={values[1]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(1, event)}/>
-                <span className="dash">-</span>
-                <input type="text" autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[2] = c} value={values[2]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(2, event)}/>
+                <input type="text" style={{width: maxLen + 'em'}} autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[0] = c} value={values[0]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(0, event)}/>
```

Comment:
Not enough to enforce element width. Overridden anyway. (tested on iPhone (5S?|SE) )
<img width="468" alt="screen shot 2016-05-12 at 08 52 34" src="https://cloud.githubusercontent.com/assets/1055844/15206411/388661f2-181f-11e6-8b78-03cd73062fab.png">